js/main.js

```diff
@@ -1,35 +1,122 @@
+		],
+	},
+	marsha: {
+		imgPath: "/assets/images/pages/team/marsha-maxwell.png",
+		name: "Dr. Marsha Maxwell, CETL ",
+		role: "Co-founder",
+		email: "info@iftheselandscouldtalk.org",
+		linkedin: "https://www.linkedin.com/in/natasha-rabsatt/",
+		bio: [
+			"With strategic, tactical and operational technology experience, Dr. Marsha Maxwell creates innovative environments and applications that harness the power of emerging technologies and neuroscience to enhance the educator's reach to better engage students and foster a culture of innovation in organizations.",
+			"Her areas of interest include Augmented Analytics, Immersive Experiences, Artificial Intelligence, Machine Learning, STEAM programs, and cybersecurity. Marsha has also been very active in bringing STEM/STEAM initiatives to life especially for underserved and underrepresented groups throughout their educational journey from kindergarten to university and beyond.",
+			"Marsha has been a featured speaker for various organizations' educational summits and workshops including Microsoft, Apple, and Google for her skills in education technology, innovation leadership, strategic analysis and decision-making, and equity and inclusion in technology.",
+		],
+	},
+};
+
+// Team Modal Functions
+
+// populate modal
+const populateTeamModal = profile => {
+	// deconstructing object
+	const { imgPath, name, role, bio, email, linkedin } = profile;
+
+	// Team modal elements
+	let memberImg = document.querySelector("#member-img");
+	let memberName = document.querySelector("#member-name");
+	let memberRole = document.querySelector("#member-role");
+	let memberEmail = document.querySelector("#member-email");
+	let memberLinkedin = document.querySelector("#member-linkedin");
+	let memberBio = document.querySelector("#member-bio");
+
+	// picture
+	memberImg.src = imgPath;
+	memberImg.alt = name;
 
-// close land acknowledgement via button
-closeLaBtn.addEventListener("click", () => {
-	landAckModal.classList.add("hidden");
+	// text
+	memberName.textContent = name;
+	memberRole.textContent = role;
+
+	// bio
+	bio.forEach(paragraph => {
+		let pElement = document.createElement("p");
+		pElement.textContent = paragraph;
+		memberBio.appendChild(pElement);
+	});
+
+	// socials
+	memberEmail.href = `mailto:${
+		email || "mailto:info@iftheselandscouldtalk.org"
+	}`;
+	memberLinkedin =
+		linkedin ||
+		"https://www.linkedin.com/company/if-these-lands-could-talk-itlct/";
+};
+
+// Global Modal Events
+// open modal button events
+openModalBtns.forEach(button => {
+	button.addEventListener("click", () => {
+		// opening modals based on data-type attr
+		let modalElement = document.querySelector(`#${button.dataset.type}`);
+
```

Comment:
We need to prevent the whole page from scrolling when the modal is open. Here is an example of how to approach this: https://www.tutorialspoint.com/how-to-prevent-body-from-scrolling-when-a-modal-is-opened-using-jquery#:~:text=Users%20can%20follow%20the%20syntax,%24('body').

Reply:
added into the js side of things with a custom class

Reply:
Works great!